atlasdb-impl-shared/src/test/resources/lockwatch-event-cache-output/leaderChangeClearsCaches/event-cache-1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "eventLog" : {
+    "snapshot" : {
+      "watches" : [ ],
+      "locked" : [ ],
+      "snapshotVersion" : null
+    },
+    "eventStore" : {
+      "eventMap" : { }
+    },
+    "latestVersion" : null
```

Comment:
Why isn't latestVersion set here?

Reply:
there has been a leader change, but has received a successful response - therefore we cannot take that response, but instead we want to treat it like a failed situation.